api/src/pcapi/core/offerers/api.py

```diff
@@ -619,7 +619,10 @@ def set_offerer_attachment_pending(
 
 
 def reject_offerer_attachment(
-    user_offerer: offerers_models.UserOfferer, author_user: users_models.User, comment: str | None = None
+    user_offerer: offerers_models.UserOfferer,
+    author_user: users_models.User,
+    comment: str | None = None,
+    with_email: bool | None = True,
```

Comment:
```suggestion
    send_email: bool | None = True,
```
je préfère, pour ne pas confondre avec une notion de colonne en bdd, mais c'est comme tu veux :)